src/.nuget/toolchain.ubuntu.14.04-x64.Microsoft.DotNet.ILCompiler.Development.nuspec

```diff
@@ -32,6 +32,8 @@
       <file src="../ToolRuntime/*.dll" />
```

Comment:
I agree. We will soon use dotnet restore and add coreclr/ryujit/objwriter/appdep as a test dependency. These are incremental changes to get our tests using CLI so E2E scenarios can be validated.